DeskKit/DeskKit/View Models/DKContactUsViewModel.m

```diff
@@ -291,9 +291,11 @@ - (NSDictionary *)caseDictionary
 - (NSDictionary *)messageDictionary
 {
     NSString *fromEmail = [self bestFromEmail];
+    NSString *body = self.bodyItem.text.string.dkTrimmedString;
+    
     NSMutableDictionary *dictionary = [@{
                                          DKMessageDirectionKey: @"in",
-                                         DKMessageBodyKey: self.bodyItem.text.string,
```

Comment:
I think it's a client's responsibility to send accurate data to the server. Also, user can enter a lot of empty spaces and send it to server. Each empty space is counted as a character and increases a payload. By truncating leading and trailing spaces, we are sending more accurate data and removing extra payload.